core/templates/dev/head/domain/exploration/StateObjectFactory.js

```diff
@@ -18,40 +18,22 @@
  */
 
 oppia.factory('StateObjectFactory', [
-  'AnswerGroupObjectFactory',
-  function(AnswerGroupObjectFactory) {
-  var State = function(name, content, interaction, paramChanges) {
+  'AnswerGroupObjectFactory', 'InteractionObjectFactory',
+  function(AnswerGroupObjectFactory, InteractionObjectFactory) {
+  var State = function(name, content, interactionBackendDict, paramChanges) {
     this.name = name;
+    this.classifierModelId = null;
```

Comment:
I don't think it's always null.... -- this should be part of the stateDict, right?